services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/progress/TransferProgressUpdater.java

```diff
@@ -107,8 +107,7 @@ public void publisherSubscribe(Subscriber<? super S3MetaRequestProgress> subscri
 
             @Override
             public void subscriberOnNext(S3MetaRequestProgress s3MetaRequestProgress) {
-                incrementBytesTransferred(Math.toIntExact(s3MetaRequestProgress.getBytesTransferred()),
-                                          s3MetaRequestProgress.getContentLength());
+                incrementBytesTransferred(Math.toIntExact(s3MetaRequestProgress.getBytesTransferred()));
```

Comment:
We probably should not cast it to int. Can we change incrementBytesTransferred to take a long instead?